modules/assim.sequential/R/sda_plotting.R

```diff
@@ -380,9 +380,9 @@ post.analysis.ggplot <- function(settings, t, obs.times, obs.mean, obs.cov, obs,
                            data.frame(mean-(sd*1.96), mean+(sd*1.96))
                            
                          })%>%
-            mutate(Variables=names(one.day.data$means))%>%
-            `colnames<-`(c('2.5%','97.5%','Variables'))%>%
-            mutate(means=one.day.data$means%>%unlist,
+            dplyr::mutate(Variables=names(one.day.data$means))%>%
+            `colnames<-`(c('`2.5%`', '`97.5%`', 'Variables'))%>%
```

Comment:
I think the check complaint about these variables happens below on line 418 -- these ones are OK to keep being just plain quoted strings.
```suggestion
            `colnames<-`(c('2.5%', '97.5%', 'Variables'))%>%
```